faircookbook_rdmkit_mapping.yml

```diff
@@ -121,4 +121,97 @@
   links:
     - fcb_title: Plant genomic and genetic variation data submission to EMBL-EBI databases
       fcb_id: FCB061
-
+      
+- rdmkit_title: Bioimaging data
+  rdmkit_filename: bioimaging_data
+  links:
+     - fcb_title: IMI EUBOPEN FAIR High-Content Screening data deposition
+       fcb_id: FCB067
+     - fcb_title: Selecting terminologies and ontologies
+       fcb_id: FCB020
+       
+- rdmkit_title: Data brokering
+  rdmkit_filename: data_brokering
+  links:
+    - fcb_title: Introduction to terminologies and ontologies
+      fcb_id: FCB019
+    - fcb_title: Creating data/variable dictionary
+      fcb_id: FCB025
+    - fcb_title: Creating a metadata profile
+      fcb_id: FCB026
+    - fcb_title: Selecting terminologies and ontologies
+      fcb_id: FCB020
+    - fcb_title: Ontology-related tools and services
+      fcb_id: FCB022
+      
+- rdmkit_title: Existing data
+  rdmkit_filename: existing_data
+  links:
+    - fcb_title: Licensing
+      fcb_id: FCB032
+       
+ - rdmit_title: Documentation and metadata
+   rdmkit_filename: metadata_management
+   links:
+     - fcb_title: Search Engine Optimization (SEO)
+       fcb_id: FCB010
+ 
+ - rdmit_title: Machine actionability
+   rdmkit_filename: machine_actionability
+   links:
+     - fcb_title: Interlinking data from different sources
+       fcb_id: FCB016
+       
+ - rdmit_title: Toxicology data
+   rdmkit_filename: toxicology_data
+   links:
+     - fcb_title: Creating InChIKeys for IUPAC names
+       fcb_id: FCB080
+ 
+ - rdmit_title: Identifiers
+   rdmkit_filename: identifiers
+   links:
+     - fcb_title: Creating InChIKeys for IUPAC names
+       fcb_id: FCB080
+     - fcb_title: Creating resolvable identifiers
+       fcb_id: FCB077
+       
+ - rdmit_title: Microbial biotechnology
+   rdmkit_filename: microbial_biotechnology
+   links:
+     - fcb_title: Selecting terminologies and ontologies
+       fcb_id: FCB020
+       
+- rdmkit_title: Data storage
+  rdmkit_filename: storage
+  links:
+    - fcb_title: Transferring data with SFTP protocol
+      fcb_id: FCB014
+    - fcb_title: Downloading data with Aspera protocol
+      fcb_id: FCB015
+    - fcb_title: Licensing Software
+      fcb_id: FCB033
```

Comment:
What is the rational for linking "Data storage" page and "Licensing Software" recipe?

Reply:
There is a specific reference to software licensing on that page:
`Determine which software you are going to use. If it is a proprietary software, you should check possible licensing issues. Check if it only runs on specific operative systems (windows, mac, linux…).`

Reply:
I am not sure about this link, but ok.